CaPPMS/Shared/ProjectEntryAuthorizedDropdown.razor

```diff
@@ -1,7 +1,9 @@
 ﻿@using CaPPMS.Model
+@using System.Windows;
```

Comment:
I think this should not be needed: @using System.Windows; Is this in use on the page?

Reply:
Removed System.Windows as required.